### ISSUE TITLE
"Master Control not up" message pops up like a bad penny on arch1900

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1009]
but I have seen it on other machines, as well. I wondered this time, whether or not it has anything to do with user privileges.
Needs a closer look at the mechanism.

**Comment by urapadmin** [comment]
I don't really see a reason why this could happen. I need more data. So I have equipped MCP itself with a new logging feature and kiosk itself will log a bit more about the missing MCP pulse.

**Comment by urapadmin** [comment]
### @test Kiosk 0.11
- [ ] The administration page in kiosk shows now if MCP is running and what version is present in system information

**Comment by urapadmin** [comment]
ja, The admin page works but the message still pops up like a bad penny. I see this behaviour only on Beset. And in the admin page it always says that MCP is up and it gives the correct version

**Comment by urapadmin** [comment]
there is a general issue with Beset and the messaging. I also got the "Housekeeping needed" message after restoring 
- late
- and then after quitting it it came back
it seems the different processes or threads have different information

**Comment by urapadmin** [comment]
in 0.11.6 I will at least make the particular check it configurable. It is too nervig.
We can now suppress the messages in the config:

```
kiosk:
  suppress_messages:
    SYS_MSG_ID_MCP_NOT_UP: false


```

**Comment by urapadmin** [comment]
yup, the whole messaging system is simply not working in a multi-process environment like Beset. I have to rethink that.

**Comment by urapadmin** [comment]
### @test kiosk 0.11.13
The messaging system has a real synchronization now and should work across processes, which means:
- [ ] If MCP is not up there should be a message
- [ ] If one user clicks that message away it should be gone - 
- [ ] until the server restarts for some reason, in which case it will come back, but with a different timestamp

**Comment by urapadmin** [comment]
it seems to work. But there is still a delay because of which it looks as if the message pops up again.
But I did not do anything when it reappeared (only if one switches to a different part of kiosk). And after a while it just disappeared.

**Comment by urapadmin** [comment]
### @test kiosk 0.12
This is now rolled out as part of the 0.12 version

**Comment by urapadmin** [comment]
reopened because of #1287

**Comment by urapadmin** [comment]
The problem is really that there can be a delay of a minute before the closing of the messages gets around to all of the running processes. I need an additional mechanism to trigger immediate message synchronization when necessary. The thing with the background thread is superfluous and can be dropped.

**Comment by luizaogs** [comment]
Closing this since #1287 is good now.

**Comment by lbestock** [comment]
Bad pennies don't go away. Running into it on the demo server (not for the first time recently, I've just forgotten to update this ticket.)

**Comment by lbestock** [comment]
Oh. Oh wait, it might be that Master Control is not up. How novel!
